### ISSUE TITLE
Initial generator pre-training

### ISSUE THREAD

**Issue by udion** [ai-tor/PyTorchSRGAN#12]
The initial generator pre-training using MSE loss is to initialize the weights properly?
